src/containerapp/azext_containerapp/_help.py

```diff
@@ -29,12 +29,6 @@
               --image nginx --environment MyContainerappEnv \\
               --cpu 0.5 --memory 1.0Gi \\
               --min-replicas 4 --max-replicas 8
-    - name: Create a container app with secrets and environment variables.
-      text: |
-          az containerapp create -n MyContainerapp -g MyResourceGroup \\
```

Comment:
Did one or both of you try this example btw to check this works?

Reply:
It works on powershell 7.x and bash 5.x

Reply:
If this works then I am ok keeping this btw.

Reply:
re-added